cabby/geo/walk.py

```diff
@@ -90,7 +129,7 @@ def get_end_poi(map: map_structure.Map) -> Optional[GeoSeries]:
     return None
 
   # Pick random POI.
-  poi = small_poi.sample(1, random_state=1).iloc[0]
+  poi = small_poi.sample(1, random_state=SEED).iloc[0]
```

Comment:
How many functions use `SEED`? If not many, I would suggest making it a function argument, and then adding it as a flag to py_binaries.

Reply:
I made a change there in another PR so I'm going to leave it for now as it is.